crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -43,6 +43,9 @@ use crate::rr::resource::RecordRef;
 
 use self::rrset::Rrset;
 
+use nsec3_validation::verify_nsec3;
```

Comment:
I'd prefer that we move `dnsssec_dns_handle.rs` to `dnssec_dns_handle/mod.rs` if we're going to do this. This helps be consistent with the way we do the other modules.